packages/integrations/tailwind/src/index.ts

```diff
@@ -14,13 +16,59 @@ function getDefaultTailwindConfig(srcUrl: URL) {
 	};
 }
```

Comment:
Similar comment here. [`tailwindcss/defaultConfig.js`](https://github.com/tailwindlabs/tailwindcss/blob/master/defaultConfig.js) exists and I think we should use it and apply our Astro-specific overrides to that.

Reply:
Hm, I'm not sure that's necessary? A config passed to the `tailwindPlugin` will be merged against Tailwind's defaults no matter what. I also apply this default config as a `preset` to avoid a bunch of spread operators against user supplied config. We could apply `getDefaultTailwindConfig` as a preset _in all cases_ if you prefer!

Reply:
Update: it looks like `defaultConfig` is accessible via `tailwindcss/tailwind-config-default`, but the type checker isn't too happy with it. It's also a _large_ JS file to import and pipe into the plugin. I feel like it's safe to keep our default as-is to cut down on this.